tests/conftest.py

```diff
@@ -25,7 +25,7 @@ def _sample_ndpi(tmpdir_factory):
 
 @pytest.fixture(scope="session")
 def _sample_svs(tmpdir_factory):
-    """Sample pytest fixture for svs images. Download ndpi image for pytest."""
+    """Sample pytest fixture for svs images. Download svs image for pytest."""
```

Comment:
"""Sample pytest fixture for svs images. 
Download svs image for pytest.

"""